client/webserver/site/cache_utilities.bash

```diff
@@ -0,0 +1,15 @@
+SITE_DIR=`cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P`
+BB_FILEPATH=${SITE_DIR}/src/html/bodybuilder.tmpl
+CSS_DIR=${SITE_DIR}/src/css
+JS_DIR=${SITE_DIR}/src/js
```

Comment:
The relevant hashes are really the outputs in dist I think.  There are other things that could change the build output without the sources changing (dependencies updated).

Reply:
Unfortunately, the output files can be either development or production builds, and our current CI flow doesn't guarantee that those files are built by the time `run_tests.sh` runs.

I've now made the cache buster a concatenation of the source dir and output hashes, but only checking the source dir hash for validation.